server/src/main/kotlin/suwayomi/tachidesk/manga/impl/util/storage/SafePath.kt

```diff
@@ -9,6 +11,10 @@ package suwayomi.tachidesk.manga.impl.util.storage
 
 // adopted from: https://github.com/tachiyomiorg/tachiyomi/blob/4cefbce7c34e724b409b6ba127f3c6c5c346ad8d/app/src/main/java/eu/kanade/tachiyomi/util/storage/DiskUtil.kt
 object SafePath {
+    fun joinPaths(vararg paths: String): String {
```

Comment:
I'm not too sure about this function, I would prefer just doing something like this instead of using a function

```kotlin
val path = Path(System.getProperty("java.io.tmpdir")) / "Tachidesk"
```

Reply:
so, just leave it as it was? https://github.com/Suwayomi/Tachidesk-Server/pull/581#discussion_r1240983810

Reply:
Path.div is an operator function for Path objects

Reply:
going a step back, does this actually matter? won't it be correct anyway once the path is used to create a File or Path?

Reply:
Nope, it doesn't matter. I just found the joinPaths function weird, if you want to revert those changes and just use / that's fine, since / is supported by every platform

Reply:
I reverted it